src/components/BackButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { ArrowBackIcon } from "@chakra-ui/icons";
+import { HStack, Button, Link } from "@chakra-ui/react";
+import { useRouter } from "next/router";
+
+export const BackButton: React.FC = () => {
+    const router = useRouter();
+
+    return (
+        <HStack spacing={6}>
+            <Link onClick={() => router.back()}>
```

Comment:
do you think you can maybe add a prop called `onClick` or `handleClick` or something, and then for the `onClick` property here you would call `props.onClick` if it exists and default to `() => router.back()`? just to make this button more reusable (e.g., in the participant info form the back button just goes to the previous form page rather than the previous url)
Thanks for making this into a component though, I did not think to do that